frontend/language/src/nb.json

```diff
@@ -1297,6 +1297,7 @@
   "ux_editor.component_properties.hasCustomFileEndings": "Egendefinerte filtyper",
   "ux_editor.component_properties.headingLevel": "Overskriftsnivå",
   "ux_editor.component_properties.hidden": "Skjul feltet",
+  "ux_editor.component_properties.hidden-help-text": "Hvis du velger å skjule et felt, kan du sette betingelser for når det skal skjules under Valg for dynamikk",
```

Comment:
I don't think we use hyphen syntax in nb json file or other places.

Reply:
fixed